android/app/src/main/kotlin/org/getlantern/lantern/model/ProUser.kt

```diff
@@ -37,10 +37,10 @@ data class ProUser(
     }
 
     val isProUser: Boolean
-        get() = isUserStatus("active")
+        get() = userLevel == "pro"
```

Comment:
Should we still check for an "active" user status (which we've traditionally used to check if a user is Pro) with 
```kotlin
isUserStatus("active") || userLevel == "pro"
```
... or is it sufficient to just check for a Pro user level now?

Reply:
I asked this query to Reflog, But its better to check both, thanks.

Reply:
Done

Reply:
Thanks! We can open another PR if need be based on his feedback